gstreamer-plugin/git-patch/0001-svthevcenc-Add-svthevc-encoder-element.patch

```diff
@@ -1898,8 +1898,6 @@ index 0000000..be1d492
 +#include <gst/video/video.h>
 +#include <gst/video/gstvideoencoder.h>
 +
```

Comment:
You do need to change line 1874: reduce 1,118 by 2. Otherwise it doesn't apply.
same for ffmpeg